File: Sample/2.0/Data/SampleContext.cs
using Microsoft.EntityFrameworkCore;
using Microsoft.EntityFrameworkCore.Metadata.Internal;

using MyApp.Models.Sample;

namespace MyApp.Data
{
  public partial class SampleContext : DbContext
  {
    public SampleContext(DbContextOptions<SampleContext> options):base(options)
    {
    }

    public SampleContext()
    {
    }



    public DbSet<Order> Orders
    {
      get;
      set;
    }

    public DbSet<OrderDetail> OrderDetails
    {
      get;
      set;
    }

    public DbSet<Product> Products
    {
      get;
      set;
    }
  }
}

File: Sample/2.0_VB/Data/SampleContext.vb
Imports Microsoft.EntityFrameworkCore
Imports Sample20VB.MyApp.Models.Sample

Namespace MyApp.Data
    Partial Public Class SampleContext
        Inherits DbContext
        Public Sub New(options As DbContextOptions(Of SampleContext))
            MyBase.New(options)
        End Sub

        Public Sub New()
        End Sub

        Public Property Orders() As DbSet(Of Order)

        Public Property OrderDetails() As DbSet(Of OrderDetail)

        Public Property Products() As DbSet(Of Product)

    End Class
End Namespace
